tools.json

```diff
@@ -12757,6 +12757,47 @@
         "RUST",
         "SCALA"
       ]
+    },
+    {
+      "name": "Hermeto",
+      "publisher": "The Hermeto Project",
+      "description": "Hermeto is a CLI tool that pre-fetches your project's dependencies, which enables network-isolated builds, which enables the production of high-accuracy SBOMs",
+      "repository_url": "https://github.com/hermetoproject/hermeto",
+      "website_url": "https://hermetoproject.github.io/hermeto/",
+      "capabilities": [
+        "SBOM"
+      ],
+      "availability": [
+        "OPEN_SOURCE",
+        "OSI_APPROVED"
+      ],
+      "functions": [
+        "AUTHOR",
+        "PACKAGE_MANAGER_INTEGRATION"
+      ],
+      "transform": [],
+      "packaging": [
+        "COMMAND_LINE_UTILITY"
+      ],
+      "library": [],
+      "platform": [
+        "LINUX"
```

Comment:
We recently merged some MAC fixes to the main branch based on an associate's MacOS dev experience. Taking my `packaging` comment above into consideration I think specifying `MAC` here as well would technically not be a lie, but this is completely optional and you can happily ignore my comment.

Reply:
Added MAC!